README.md

```diff
@@ -7,3 +7,59 @@ first line of code to publishing their first project.
 ## Website
 
 Check out the `gh-pages` branch for Hack Camp's website.
+
+## Objective
+
+The objective of Hack Camp is build an set of materials that allows any hackEDU
+team member to reproduce an experience that that brings people from no coding
+knowledge to having the ability, motivation, and confidence to create coding
+projects and continue learning new coding concepts with others, without the
+explicit urging or leading of a teacher.
+
+This can be broken down into two parts: creating the experience and reproducing
+it.
+
+#### Cohorts
```

Comment:
Yeah – that was intentional for formatting reasons. Slightly easier to read with a smaller heading in the README. Do you think that's alright?